numpyro/distributions/continuous.py

```diff
@@ -451,32 +451,45 @@ def tree_flatten(self):
 class Gompertz(Distribution):
     """Gompertz Distribution sampling utilities.
 
-    All derivations can be found in this paper (Lenart, 2012):
-        https://doi.org/10.1080/03461238.2012.687697
-    with the exception that we call the parameters `concentration` and `rate`,
-    as opposed to their "location" and "scale" terms.
+    This implementation is based off the derivations from Wikipedia page for the Gompertz distribution.
```

Comment:
Could you make the doc max line length shorter? Currently, some lines are long and `lint` does not pass.